README.md

```diff
@@ -1,2 +1,29 @@
-# Wayang Website
-Apache Wayang is the API for big data and data processing across multiple data processing frameworks. Wayang enables data engineers, data scientists and data operations to use multiple platforms at the same time. The repo contains the website for [Apache Wayang (incubating)](https://wayang.apache.org)
+# Apache Fury (incubating) Website
```

Comment:
It's referencing the Fury website

Reply:
hmm, its not in my local repo. I close this PR since it's too messed, the other one is a clean sync into an orphan branch (docusaurus).